ksql-cli/src/main/java/io/confluent/ksql/cli/Cli.java

```diff
@@ -118,10 +122,14 @@ private static void validateClient(
     try {
       RestResponse restResponse = restClient.makeRootRequest();
       if (restResponse.isErroneous()) {
+        KsqlErrorMessage ksqlError = restResponse.getErrorMessage();
+        if ((ksqlError.getErrorCode() / Errors.HTTP_TO_ERROR_CODE_MULTIPLIER)
```

Comment:
Suggest that `(ksqlError.getErrorCode() / Errors.HTTP_TO_ERROR_CODE_MULTIPLIER)` should be a method call e.g. `Errors.toStatusCode(ksqlError.getErrorCode)`